### PR TITLE
Create why-https.md

### PR DESCRIPTION
**Pull request by osvaldoabel**
Translating this article to brazilian portuguese

Address of the issue https://github.com/google/WebFundamentals/issues/3851

The name I used to sign the CLA is Osvaldo Abel.

### CODE REVIEW

**Comment by osvaldoabel** [comment]
I signed it

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="9WuP4KcDBpI"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+Você deve sempre proteger todos os seus sites com HTTPS, mesmo que eles não
+trabalhem com comunicações sensíveis. Além de oferecer segurança crítica e integridade dos dados
+do seus sites e das informações pessoais de seus usuários, o HTTPS
+Um requisito para muitos recursos dos navegador mais recentes, particularmente aqueles necessários
```
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: "... HTTPS é um ..."

Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: "...dos navegadores..."

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+trabalhem com comunicações sensíveis. Além de oferecer segurança crítica e integridade dos dados
+do seus sites e das informações pessoais de seus usuários, o HTTPS
+Um requisito para muitos recursos dos navegador mais recentes, particularmente aqueles necessários 
+em [aplicativos web progressivos](/web/progressive-web-apps/).
+
+### TL;DR {: .hide-from-toc }
+
+* Tanto intrusos malignos quanto benignos exploram todos os recursos desprotegidos entre seus sites e os usuários.
+* Muitos intrusos analisam uma série de comportamentos para identificar seus usuários.
+* O HTTPS não bloqueia apenas o mau uso do seu site. Ele Também é um requisito para muitos recursos de ponta e uma tecnologia habilitadora para recursos semelhantes a aplicativos, como os manipuladores de serviços.
```
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: "Também" -> "também"

Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: I'm not sure if "service worker" should be translated. Is it translated elsewhere? It sounds weird.

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## O HTTPS protege a integridade do teu website
```
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: "teu" -> "seu"

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+
+O HTTPS ajuda a impedir que intrusos alterem a comunicação
+Entre seus sites e os navegadores de seus usuários. O Intruso inclue
```
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: "Entre" -> "entre"
"Intruso" -> "intruso"
"inclue" -> "inclui"

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+Intencionalmente atacantes maliciosos, e as empresas legítimas, mas intrusivas,
```
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: Don't use upper case when you don't need to.

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md (hunk 4 above)
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: This comment, "O Intruso inclue..." is weird. The original comment is to describe what "intruders" means, by giving specific examples of intruders. The way the translation reads, it sounds like "the intruder intentionally includes attacks". I would translate as "Intrusos podem ser agressores intencionalmente maliciosos ou empresas legítimas mas intrusivas, como provedores de internet e hotéis que injetam anúncios em páginas."

File: src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+Como ISPs ou hotéis que injetam anúncios em páginas.
+
```
Comment (mangini) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: I stopped reviewing here, but general comments:
- Remember the rules for upper-case. You are using them where you can't.
- Decide if you use "teu" or "seu" and use it consistently.
- Be careful with number conjugation (using plural where it should be singular and vice-versa)
- Read your text after translating and check that you are not doing literal translation. When it happens, it sounds awful.

Comment (osvaldoabel) [inline src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md]: Ok, @mangini. I will review it.
Thanks :)

**Comment by petele** [comment]
Closing for now. Feel free to re-open once the issues have been addressed.

**Pull request closed without merging**